### ISSUE TITLE
TracerUtil

### ISSUE THREAD

**Issue by TornjV** [parkjs814/AlgorithmVisualizer#94]
```
refineByType: function (item) {
  return $.isNumeric(item) ? this.refineNumber(item) : this.refineString(item);  
},
refineString: function (string) {
    return string === '' ? ' ' : string;
},
refineNumber: function (number) {
    return number === Infinity ? '∞' : number;
}
```

Jquery .isNumeric(Infinity) returns false, so shouldn't it be checked in refineString ?

**Comment by nem035** [comment]
Yup, you're correct. I'll make a quick patch.

**Comment by nem035** [comment]
Solved by #95
